roles/matrix-bridge-mx-puppet-skype/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+- import_tasks: "{{ role_path }}/tasks/init.yml"
+  tags:
+    - always
+
+- import_tasks: "{{ role_path }}/tasks/validate_config.yml"
+  when: "run_setup|bool and matrix_mx_puppet_skype_enabled|bool"
+  tags:
+    - setup-all
+    - setup-puppet-skype
```

Comment:
Maybe it's more consistent with the other bridges if we call these `setup-mx-puppet-skype`?

Reply:
Fixed